open-models/use-cases/guess_app.ipynb

```diff
@@ -0,0 +1,816 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "8d836e0210fe"
+      },
+      "outputs": [],
+      "source": [
+        "from huggingface_hub import interpreter_login\n",
+        "\n",
+        "interpreter_login()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "c71a4314c250"
+      },
+      "source": [
+        "Read more about [Hugging Face Security](https://huggingface.co/docs/hub/en/security), specifically about [Hugging Face User Access Tokens](https://huggingface.co/docs/hub/en/security-tokens)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "DF4l8DTdWgPY"
+      },
+      "source": [
+        "### Set Google Cloud project information and initialize Vertex AI SDK\n",
+        "\n",
+        "To get started using Vertex AI, you must have an existing Google Cloud project and [enable the Vertex AI API](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com), if not enabled already.\n",
+        "\n",
+        "Learn more about [setting up a project and a development environment](https://cloud.google.com/vertex-ai/docs/start/cloud-environment)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Nqwi-5ufWp_B"
+      },
+      "outputs": [],
+      "source": [
+        "import os\n",
+        "\n",
+        "from google.cloud import aiplatform\n",
+        "\n",
+        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\", isTemplate: true}\n",
+        "if PROJECT_ID == \"[your-project-id]\":\n",
+        "    PROJECT_ID = str(os.environ.get(\"GOOGLE_CLOUD_PROJECT\"))\n",
+        "\n",
+        "LOCATION = os.environ.get(\"GOOGLE_CLOUD_REGION\", \"us-central1\")\n",
+        "\n",
+        "aiplatform.init(project=PROJECT_ID, location=LOCATION)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ee37e1544281"
+      },
+      "source": [
+        "### Requirements"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "877cd3fb2dce"
+      },
+      "source": [
+        "To successfully deploy your Hugging Face model on Vertex AI, you need to have the following IAM roles set:\n",
+        "\n",
+        "- Artifact Registry Reader (roles/artifactregistry.reader)\n",
+        "- Vertex AI User (roles/aiplatform.user)\n",
+        "\n",
+        "For more information about granting roles, see [Manage access](https://cloud.google.com/iam/docs/granting-changing-revoking-access).\n",
+        "\n",
+        "---\n",
+        "\n",
+        "You also need to enable the following APIs (if not enabled already):\n",
```

Comment:
I'd recommend adding a `gcloud` command to enable those APIs.

Reply:
I listed APIs because not always devs have permissions to enable them using gcloud